src/test/resources/BulkCopyCSVTestInputDelimiterEscape.csv

```diff
@@ -9,3 +9,7 @@
 9@1997@Ford@E350@"Super@ ""luxurious"" truck"@ 
 10@1997@ "Ford" @E350@  "E63"@ 
 11@1997@Ford@E350@" Super luxurious truck "@ 
+12@1997@"F
+o
+r
+d"@"E350"@"""Super"" ""luxurious"" ""truck"""@
```

Comment:
needs new line here

Reply:
Where would you like it?  Are you wanting to break up the "Super" "luxurious" "truck" field?

Reply:
I mean at the very end of the file where you see the red mark

Reply:
Ok, sure.